src/docs/05_building_block_view.adoc

```diff
@@ -57,17 +57,52 @@ In the best case you will get away with examples or simple signatures.
 
 ****
 
-_**<Overview Diagram>**_
+[plantuml,"Whitebox overall system",png]
+----
+node ViaDe {
+    [Solid module] as solidmod
+    [Map module] as map
+    [Route interpretation] as routes
+    [React] as react
+}
+
+node "Solid server" {
+    database "User's POD" as pod
+}
+
+node "Maps service" {
+    () "Maps API" as imaps
+}
+
+:User: --> react
+react --> map
+map --> routes
+map --> imaps
+routes --> solidmod
+react --> solidmod
+solidmod --> pod
+----
```

Comment:
It crashes!
![crash](https://user-images.githubusercontent.com/24551117/74726251-6c343700-523f-11ea-9d6b-d666de37297f.JPG)

Reply:
@garciaAlvarezPelayo @alvarezGarciaMarcos @alvarezGarciaMarcos if have the same problem, install: https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.msi

Reply:
The solution to this problem was found: the dependency Graphviz was missing locally.